RFCs/2021-04-16-76-collection-policies.md

```diff
@@ -84,8 +84,12 @@ visor:
       description: "base chaning NET to DNS policy"
       # input stream to create based on the given tap and optional filter config
       input:
-        # this must reference a tap name, or application of the policy will fail
-        tap: anycast
+        # this must reference valid tags existent on applied taps, or application of the policy will fail
+        tap_selector:
+          # It can be either "any" or "all"
```

Comment:
i think we probably want an optional `input_type` here to ensure a tap_selector only matches tags against a certain input source, if that makes sense for this policy. it may not always - for example, a dns policy that should work on both pcap and dnstap. but you may have flow policy that should never match against anything but flow input.

Reply:
https://github.com/ns1labs/pktvisor/blob/8e41c48e716836d6bfe97bdc132f6bc38354e37b/src/Policies.cpp#L226

Reply:
cool